index.html

```diff
@@ -0,0 +1,1975 @@
+              <div class="deal d-flex">
+                <img alt="deal"
+                  src="https://cdnp3.stackassets.com/600275c367415b9343a9b4812472fa05f7db59a8/store/dd972578f03549a52ceaac3644e38a3ee41cc83aef912816935d031e21d2/sale_14198_primary_image.jpg">
+                <div class="deal-info">
+                  <h4>
+                    <a href="#" class="black">The Ultimate Microsoft Excel Certification Training Bundle</a>
+                  </h4>
+                  <div class="deal-price d-flex">
+                    <span class="old as-center">$945</span>
+                    <span class="new as-center">$33.99</span>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="second-row-deals">
+            <div class="li-div-n">
+              <div class="deal d-flex">
+                <img alt="deal"
+                  src="	https://cdnp0.stackassets.com/825637132390d0153f93083ff14920c7cab3f6a8/store/0b7aed89b153b14123f3957f71bdc6f9cfa6a80d601430accf84b8fe2f41/sale_38865_primary_image.jpg">
+                <div class="deal-info">
+                  <h4>
+                    <a href="#" class="black">Online Income 101: Freelancing, Influencer Marketing, Onlince Courses and
+                      More</a>
+                  </h4>
+                  <div class="deal-price d-flex">
+                    <span class="old as-center">$2600</span>
+                    <span class="new as-center">$59.99</span>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="li-div-n">
+              <div class="deal d-flex">
+                <img alt="img"
+                  src="https://cdnp0.stackassets.com/5cb5497e340bdf203caad36842570c00d492443c/store/4cf3c3c26932a65b8b43b7e6e2d7c1609800afde93a16ace92d0c4ac102d/sale_37002_primary_image.jpg">
+                <div class="deal-info">
+                  <h4>
+                    <a href="#" class="black">The Ultimate Design Assets Bundle: Lifetime Subscription</a>
+                  </h4>
+                  <div class="deal-price d-flex">
+                    <span class="old as-center">$12198</span>
+                    <span class="new as-center">$49.99</span>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="li-div-n">
+              <div class="deal d-flex">
+                <img alt="deal"
+                  src="https://cdnp3.stackassets.com/5520ced2e2087194c396763c173fc619db5a52a3/store/435968f3d598af6f168bca2a38af51e83929a031333582e96a49cea4fa6e/sale_29727_primary_image.jpg">
+                <div class="deal-info">
+                  <h4>
+                    <a href="#" class="black">BetterMe Home Workout & Diet: Lifetime Subscription</a>
+                  </h4>
+                  <div class="deal-price d-flex">
+                    <span class="old as-center">$1200</span>
```

Comment:
- [ ] The layout in the tablet view needs to be 3 articles per columns.

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/93134391-1decf680-f6af-11ea-8690-2508ab7c7ab6.png) | ![image](https://user-images.githubusercontent.com/61479648/93134473-38bf6b00-f6af-11ea-92ad-1fc53e20bf9d.png) |